Servers/Administration/OJS.Servers.Administration/Controllers/BaseAutoCrudAdminController.cs

```diff
@@ -22,4 +24,27 @@ protected override IEnumerable<string> HiddenFormControlNamesOnCreate
         {
             nameof(IEntity<object>.Id),
         };
+
+    protected int GetEntityIdFromQuery(IDictionary<string, string> complexId)
```

Comment:
Won't this work only for entities with a single primary key?

Reply:
It could be extended, if there is a need for a complex primary key.

Reply:
Leave a comment, for refference